src/coreclr/inc/corinfo.h

```diff
@@ -720,6 +720,31 @@ enum class CorInfoCallConvExtension
 };
 
 #ifdef TARGET_X86
+inline const char *GetCallConvName(CorInfoCallConvExtension callConv)
```

Comment:
This code shouldn't be in the "interface" header function. It looks like it's only used by debug code in the JIT, so should be defined in the JIT, only under `#ifdef DEBUG`

Reply:
Thanks!